part09-maas-provider/main.tf

```diff
@@ -0,0 +1,4 @@
+resource "maas_instance" "mynode" {
+  release_erase = false  
+  release_erase_quick = true
+}
```

Comment:
I think that it is possible to add `maas_interface_link` as another resource and using `data` block to fetch some information.